presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/PrestoExceptionClassifier.java

```diff
@@ -128,7 +130,13 @@ public static Builder defaultBuilder()
                 // From ThriftErrorCode
                 .addRetryableError(THRIFT_SERVICE_CONNECTION_ERROR)
                 // Conditional Retryable Errors
-                .addRetryableError(EXCEEDED_TIME_LIMIT, Optional.of(DESCRIBE), Optional.empty());
+                .addRetryableError(EXCEEDED_TIME_LIMIT, Optional.of(DESCRIBE), Optional.empty())
```

Comment:
I think line 133-139 should belong to the next commit

Reply:
This commit is refactoring by converting `shouldResubmit` from a static method to an instance method. The behavior of the function should not change.

Before, this commit, `shouldResubmit` would return true for certain cases, including  `HIVE_PARTITION_DROPPED_DURING_QUERY `, `HIVE_TABLE_DROPPED_DURING_QUERY`, and "table already exists" errors. Line 134 - 139 is only to keep the behavior unchanged for this refactoring.